client/verta/verta/client.py

```diff
@@ -138,7 +140,7 @@ def __init__(self, host=None, port=None, email=None, dev_key=None,
 
         if host is None:
             raise ValueError("`host` must be provided")
-        auth = {_utils._GRPC_PREFIX+'source': "PythonClient"}
+        auth = {**extra_auth_headers, _utils._GRPC_PREFIX+'source': "PythonClient"}
```

Comment:
Thank you for the contribution @lsb!

Unpacking inside a `dict` literal is unfortunately a syntax error in Python 2, which we'd like to maintain support for where possible; I believe this alternative should work:
```suggestion
        auth = extra_auth_headers.copy()
        auth.update({_utils._GRPC_PREFIX+'source': "PythonClient"})
```